config.json

```diff
@@ -30,7 +30,8 @@
       "one_page_check_out",
       "product_videos",
       "google_amp",
-      "customized_checkout"
+      "customized_checkout",
+      "account_payment_methods"
```

Comment:
This will need an entry in theme registry features list as well. See https://github.com/bigcommerce/theme-registry/blob/master/app/schemas/tag.json#L2 and here https://github.com/bigcommerce/stencil-cli/blob/e5c3e96e3a3d10e4423afec418c2c8f3686d2d02/lib/themeConfig.schema.json#L33 as well.

Reply:
You need this bcoz for marketplace theme we validate against a fixed set of values.